src/components/Search/SearchComponent.tsx

```diff
@@ -101,12 +96,13 @@ function SearchComponent({
           </Tooltip>
         </InputLabel>
         {/**** Edit Button ****/}
-        {hasNonEditableState && !formIsDisplayed && (
+        {/* temporary until global edit button PR */}
+        {hasNonEditableState && !formIsDisplayed && isBaseComp && (
```

Comment:
nit: I'd rather have a boolean prop `showEditButton`

Reply:
Yes, the name has been changed in my last PR. I'm going to handle the edit button in a new PR since this one is already crowded.